ush/test/config/bufr_script4backend_satwnd_amv_ahi.yaml

```diff
@@ -15,7 +15,7 @@ observations:
         args:
           input_path: "./testinput/2021080100/gdas.t00z.satwnd.tm00.bufr_d"
           mapping_path: "./bufr_satwnd_amv_ahi_mapping.yaml"
-          category: "h8"
+          category: ["h8"]
```

Comment:
I looked at the python code and it looks like it takes strings (converts them to lists). So you probably don't need these changes. 

Sorry for my confusion...